src/app/vendor/videojshooks.js

```diff
@@ -458,51 +461,7 @@ var suggestedExternal = function () {
     } catch (e) {}
     return link;
 };
-vjs.ErrorDisplay.prototype.update = function () {
```

Comment:
Can you explain, why this block (and the following) should be removed please ?

Reply:
They're both duplicate of code that is a bit higher in the file. So there's no change in functionality or anything just some housework.

Reply:
oh ok, my bad